sui-execution/latest/sui-move-natives/src/object_runtime/object_store.rs

```diff
@@ -329,6 +346,16 @@ impl<'a> Inner<'a> {
                 ),
             ),
         };
+        // save a fingerprint
+        let fingerprint =
+            ObjectFingerprint::original(protocol_config, &parent, child_move_type, &v).map_err(
+                |e| {
+                    PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR).with_message(
```

Comment:
what happens if this errors fires?

Reply:
It is an invariant violation. No different than any other VM invariant violation?